src/filesystem.h

```diff
@@ -65,7 +71,7 @@ int64_t Fs_Load(const char *filename, void **buffer);
 void Fs_Free(void *buffer);
 _Bool Fs_Rename(const char *source, const char *dest);
 _Bool Fs_Unlink(const char *filename);
-void Fs_Enumerate(const char *pattern, Fs_EnumerateFunc, void *data);
+int32_t Fs_Enumerate(const char *pattern, Fs_EnumerateFunc, void *data);
```

Comment:
I feel like the return type here could be either FsEnumerateResult or _Bool.

Reply:
_Bool is preferable, yeah. I just picked int32_t because that's what PhysFS returns, but it only returns 0 or 1.